website/docs/r/connect_queue.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Connect"
+layout: "aws"
+page_title: "AWS: aws_connect_queue"
+description: |-
+  Provides details about a specific Amazon Connect Queue
+---
+
+# Resource: aws_connect_queue
+
+Provides an Amazon Connect Queue resource. For more information see
+[Amazon Connect: Getting Started](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-get-started.html)
+
+~> **NOTE:** Due to The behaviour of Amazon Connect you cannot delete queues.
```

Comment:
Nice job including this note for the users. Having thorough documentation is important.